contracts/Gateway.sol

```diff
@@ -57,6 +57,9 @@ contract Gateway is ProtocolVersioned, Owned {
       uint256 _unlockHeight,
       bytes32 _stakingIntentHash);
 
+    /** Below event is emitted after successful execution of setWorkers */
+    event WorkersSet(bytes32 _uuid, WorkersInterface _workers);
```

Comment:
I don't believe `_uuid` is needed

Reply:
I have removed it.

I am not aware of the use-case where this will be consumed. I believe that when block scanner will read the receipt, it will get the information about the Gateway address and workers address.  UUID was an additional info that was available to extract from the receipt rather than going to external DB.

So it make sense to not have it for now until we get a use-case for this